src/views/AddItem.jsx

```diff
@@ -109,57 +118,113 @@ export function AddItem({ listToken, data }) {
 
 	return (
 		<>
-			<Form onSubmit={onFormSubmit}>
-				<Form.Label htmlFor="item">Item Name:</Form.Label>
-				<InputGroup>
-					<Form.Control
-						type="text"
-						id="item"
-						name="item"
-						value={itemName}
-						onChange={(e) => setItemName(e.target.value)}
-						required={true}
-					/>
-				</InputGroup>
-				<Form.Label id="purchase-date-label">
-					How soon will you buy this again?
-				</Form.Label>
-				<fieldset>
-					<div>
-						<Form.Check
-							defaultChecked
-							aria-labelledby="purchase-date-label"
-							type="radio"
-							value="soon"
-							id="soon"
-							name="radio-btn"
-							onChange={handlePurchaseDate}
-							label="Soon"
-						/>
-					</div>
-					<div>
-						<Form.Check
-							type="radio"
-							value="kind-of-soon"
-							id="kind-of-soon"
-							name="radio-btn"
-							onChange={handlePurchaseDate}
-							label="Kind of Soon"
-						/>
-					</div>
-					<div>
-						<Form.Check
-							type="radio"
-							id="not-soon"
-							value="not-soon"
-							name="radio-btn"
-							onChange={handlePurchaseDate}
-							label="Not Soon"
-						/>
-					</div>
-				</fieldset>
-				<Button type="submit">Add Item</Button>
-			</Form>
+			<div className="add-item">
+				<header className="add-item-header">
+					<h1>Add Item</h1>
+				</header>
+
+				<Container className="add-item-form">
+					<Form onSubmit={onFormSubmit}>
+						<div className="form-group">
+							<Form.Label htmlFor="item">Item Name:</Form.Label>
+							<InputGroup>
+								<Form.Control
+									type="text"
+									id="item"
+									name="item"
+									value={itemName}
+									onChange={(e) => setItemName(e.target.value)}
+									required={true}
+									placeholder="e.g., croissants"
+									className="item-input"
+								/>
+							</InputGroup>
+						</div>
+						<div className="form-group">
+							<Form.Label id="purchase-date-label">
+								How soon will you buy this again?
+							</Form.Label>
+							<fieldset>
+								<Row className="soon-row purchase-date-row">
```

Comment:
I'm not too familiar with `Row` and `Col` specifically, but I'm glad you're able to utilize them here!